test/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -36,16 +41,32 @@
 import org.sagebionetworks.bridge.models.subpopulations.Subpopulation;
 import org.sagebionetworks.bridge.models.subpopulations.SubpopulationGuid;
 
+import org.sagebionetworks.client.SynapseClient;
+import org.sagebionetworks.client.exceptions.SynapseException;
+import org.sagebionetworks.reflection.model.PaginatedResults;
+import org.sagebionetworks.repo.model.AccessControlList;
+import org.sagebionetworks.repo.model.Entity;
+import org.sagebionetworks.repo.model.MembershipInvtnSubmission;
+import org.sagebionetworks.repo.model.Project;
+import org.sagebionetworks.repo.model.ResourceAccess;
+import org.sagebionetworks.repo.model.Team;
+import org.sagebionetworks.repo.model.util.ModelConstants;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import com.google.common.collect.Sets;
 
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 @ContextConfiguration("classpath:test-context.xml")
 @RunWith(SpringJUnit4ClassRunner.class)
 public class StudyServiceTest {
 
+    private static final Long TEST_USER_ID = Long.parseLong("3348228"); // test user exists in synapse
```

Comment:
Another reason to put this in config is perhaps it'll be different if we switch Synapse environments, e.g. if we use their staging servers for our dev/uat environments, and another user in Synapse production for our production system. (Actually I think they copy the DBs back and forth, but just in case this ever changes, don't want it to be a constant in the code somewhere).